web-common-marlowe/src/Marlowe/Semantics.purs

```diff
@@ -797,6 +803,15 @@ newtype State
   { accounts :: Accounts
   , choices :: Map ChoiceId ChosenNum
   , boundValues :: Map ValueId BigInteger
+  -- The minSlot is a lower bound for the current slot. When we are in the context of a Wallet or Dashboard
+  -- we can just ask the time and calculate the current blockchain slot. But when we are in the context
+  -- of a running contract we can't know the exact slot as transactions only provide an interval
+  -- [lowSlot, highSlot].
+  -- So the minSlot is the maximum number of the lowSlot we had so far, and we know that the current slot is
+  -- higher than that (because slots don't go back in time).
```

Comment:
Sorry, I think it is "higher or equal than that"

Reply:
👍🏻